app/views/workspaces/_workspace.html.erb

```diff
@@ -1,7 +1,18 @@
-<li>
-  <%= workspace.name %>
-  <% if in_edit_mode %>
-    <%= link_to 'Show', workspace %>
-    <%= link_to 'Destroy', workspace, method: :delete, data: { confirm: 'Are you sure?' } %>
-  <% end %>
-</li>
+<div class="col-4">
+  <div class="grid-item">
+    <h3><%= link_to workspace.name, workspace %></h3>
```

Comment:
Do we also need to truncate the title here? It doesn't seem to wrap?

![screen shot 2017-04-10 at 1 43 38 pm](https://cloud.githubusercontent.com/assets/1656824/24874766/cd700d56-1df3-11e7-9e53-4c314afce384.png)